src/core/controller.ts

```diff
@@ -1,13 +1,10 @@
 import { Handler, MiddlewareHandler } from "hono";
-import { Bindings, DBConfig, Key, User } from "./type";
-import { uuid } from "@cfworker/uuid";
+import { Bindings, DBConfig, Key, User } from "./type.ts";
 import { StatusCode } from "hono/utils/http-status";
 
 export const users: Record<string, Handler<{ Bindings: Bindings }>> = {
   async init(ctx) {
-    const { metadata: user } = await ctx.env.OPENCAT_DB.getWithMetadata(
-      "user::id::0"
-    );
+    const { value: user } = await opencatDB.get<User>("user::id::0");
```

Comment:
Can we use array-style keys (`["user", "id", 0]`) as the common abstraction? It has the advantage of, on a supported platform (Deno), numeric parts of the key are sorted in the numeric order, i.e. you get `0, 1, 2, ... 10` when listing the keys instead of `0, 1, 10, 2`).

Reply:
Yes, this is a good suggestion. Thank you! I will change it.